### ISSUE TITLE
Error, when I use jcf.file

### ISSUE THREAD

**Issue by dshm** [w3co/jcf#5]
I have next code

 ```
<input type="file"  class="jcf-file" data-jcf='{"buttonText": "File", "placeholderText": "Add file"}'>
```

```
<script src="js/vendor/jcf.js"></script>
<script src="js/vendor/jcf.file.js"></script>
<script src="js/vendor/jcf.scrollable.js"></script>
```
```
(function ($) {
if ($('.jcf-file').exist()) { //true
    jcf.replaceAll();
}

__if I use this code i have an error ''Uncaught TypeError: jcf.replaceAll is not a function"__
__ but scrollable and select work normal __
__Can you help me?__

**Comment by inferpse** [comment]
Hi!

Your code looks correct and it seems that there are no errors. What version of JCF library do you use? Please check that you don't use multiple JCF libraries in your project.

BTW, do you use `require.js` in your project? There was a problem with AMD modules which was fixed in #2 . That problem might lead to such behavior.

Could you provide http://jsfiddle.net/ so the issue could be reproduced?

**Comment by dshm** [comment]
Sorry, I use multiple JCF libraries...Another developer has implemented a script to other script file.
Thanks for help)
P.S.
Спасибо))
